tests/test.py

```diff
@@ -0,0 +1,43 @@
+from unittest.mock import patch
```

Comment:
Please add another test file
Fetches all the models and test all of them!
We also need CI for this as well.